rskj-core/src/test/java/co/rsk/peg/federation/FederationChangeIT.java

```diff
@@ -414,8 +556,130 @@ private void assertNewAndOldFederationsHaveExpectedAddress(
         assertEquals(expectedOldFederationAddress, bridgeSupport.getRetiringFederationAddress());
     }
     
-    private void assertFundsWereNotMigrated() throws Exception {
+    private void assertMigrationHasNotStarted() throws Exception {
```

Comment:
I think this method is missing some assertion to match what the method's name means about what it is doing. I think, to assert the migration has not started, it should check:

- Current block is behind `fedActivationAge + fundsMigrationAgeBegin`
- No existing retiring fed utxos has been transferred to the new fed. For this, it should check retiring fed utxos still the same, and there is no active fed utxo.

Wdyt?

Reply:
I have added the first bullet point. I believe the second is already tested in `assertUTXOsReferenceMovedFromNewToOldFederation`. Is that correct?